### PR TITLE
rgwy40, rgwy152 - add notifier CLI and json fix

### PR DESCRIPTION
**Pull request by vjgene**
CLI to perform basic notifier operations

### CODE REVIEW

File: notifier-cli/Notifier.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        else:
+            self.startStop('run')
+
+    def stop(self):
+        try:
+            pid = None
+            if os.path.isfile("application.pid"):
+                with open("application.pid", "r") as f:
+                    pid = f.read()
+                resp = subprocess.run(['kill', pid])
```
Comment (artem-brazhnikov) [inline notifier-cli/Notifier.py:51]: is there a way to stop the service gracefully?

File: notifier-cli/Notifier.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                    args= args + "-D" + x + "=" + str(self.config.configJson[x]) + " "
+                args = args + '"'
+                os.system(self.mvn + " spring-boot:" +cmd + " " + args)
+            except Exception as e:
+                print(e)
+                print("failed to start notifier")
+
+    def healthCheck(self):
+        try:
+            response = requests.get("http://127.0.0.1:8080/actuator/health", verify=False)
```
Comment (artem-brazhnikov) [inline notifier-cli/Notifier.py:72]: I believe there is a more effective way to access the url available. Let's save the address and port separately in a var, or 2 vars and combine the url here

File: notifier-cli/README.MD [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Python3.6 or later
+
+Pip
+
+### Configure
+```notifier-cli/install.sh```
+
+### Run
+
+ ```notifier-cli/cli.sh```
```
Comment (artem-brazhnikov) [inline notifier-cli/README.MD:13]: let's copy the script into a binary folder that is included in the PATH variable so a person can run the script as an application `notifier-cli`

**Review by artem-brazhnikov — CHANGES_REQUESTED** [review]
Overall looks good for a first version

File: notifier-cli/Notifier.py (hunk 1 above)
Comment (vjgene) [reply notifier-cli/Notifier.py:51]: Actually the other way requires exposing the endpoint /actuator/shutdown which will need to be secured. so, we have to add authentication to the application in order to achieve that. Some endpoints in our app requires to work without authentication for ex. getSubscriptionPlans.

File: notifier-cli/Notifier.py (hunk 2 above)
Comment (vjgene) [reply notifier-cli/Notifier.py]: This is fixed in new commit

File: notifier-cli/README.MD (hunk 3 above)
Comment (vjgene) [reply notifier-cli/README.MD]: notifier-cli command copied to /usr/local/bin

File: notifier-cli/Notifier.py (hunk 1 above)
Comment (artem-brazhnikov) [reply notifier-cli/Notifier.py:52]: ok good

**Review by artem-brazhnikov — APPROVED** [review]
looks good, during this week I'll download and try using it

**Pull request merged**